assets/stylesheets/shared/mixins/_elevation.scss

```diff
@@ -0,0 +1,38 @@
+// A mixin to make it easy to add Material elevation
+// to elements with CSS.
+
+@mixin elevation ( $dp ) {
+    @if $dp==0 {
+        box-shadow: none
+    }
+    @else if $dp==1 {
+        box-shadow: 0 1px 1px 0 rgba(0, 0, 0, .14), 0 2px 1px -1px rgba(0, 0, 0, .12), 0 1px 3px 0 rgba(0, 0, 0, .20)
```

Comment:
```suggestion
        box-shadow: 0 1px 1px 0 rgba( var( --color-neutral-900 ), .14), 0 2px 1px -1px rgba( var( --color-neutral-900 ), .12), 0 1px 3px 0 rgba( var( --color-neutral-900 ), .20)
``

This will keep us using variables. We don't currently have a `black` variable but we have a `white` one. Do you think we should have a black one for cases like this?

Reply:
Having a `--color-black` seems obvious to me. I honestly expected that we did already.